engine/gamesys/src/gamesys/scripts/script_label.cpp

```diff
@@ -204,6 +199,12 @@ namespace dmGameSystem
  * end
  * ```
  */
+
+
+    // As seen in gamesys_private.h (which makes it a _lot_ harder to search for)
+    static const char* LABEL_EXT = "labelc";
+    //static const dmhash_t LABEL_EXT_HASH = dmHashString64(LABEL_EXT);
```

Comment:
junk?

Reply:
well, I left it as "documentation", but perhaps I should just remove it